.env.example

```
@@ -0,0 +1,25 @@
+# Prefix all client variables with NEXT_PUBLIC_
+
+# Road Variables (Prefix ROAD_)
+NEXT_PUBLIC_ROAD_WIDTH
```

dotenv-linter (4.0.0)

[warning] [KeyWithoutValue] The NEXT_PUBLIC_ROAD_WIDTH key should be with a value or have an equal sign

(KeyWithoutValue)